### ISSUE TITLE
Add [defense name]

### ISSUE THREAD

**Issue by tabrisweapon** [fra31/auto-attack#40]
**Paper**: { http://arxiv.org/abs/2010.01279}

**Venue**: {unpublished}

**Dataset and threat model**: {CIFAR-10, l-inf, eps=8/255, AutoAttack}

**Code**: { for testing, please refer to https://github.com/tabrisweapon/A-temp-project}

**Pre-trained model**: {https://www.dropbox.com/s/89i5zoxa2ugglaq/wrn-34-15-cad59.pt?dl=0}

**Log file**: {None}

**Additional data**: {yes}

**Clean and robust accuracy**: {clean:87.67%, AutoAttack: 60.65%}

**Architecture**: {WideResNet-34-15}

**Description of the model/defense**: 
’‘’
Dear authors of AutoAttack:
  This is an update for our last submission in https://github.com/fra31/auto-attack/issues/21. Here we report our new best results and hope to replace it with the current one on the table (the 4th one). We also change our title from "Does Network Width Really Help Adversarial Robustness?" to "Do Wider Neural Networks Really Help Adversarial Robustness?". Please update this information for us on the RobustBench too.

Thanks!
Boxi Wu
‘’‘

**Comment by fra31** [comment]
Hi,

thanks for sharing your new model! I've updated the entry. I'll also change the title on RobustBench.

**Comment by tabrisweapon** [comment]
> Hi,
> 
> thanks for sharing your new model! I've updated the entry. I'll also change the title on RobustBench.

Thanks!
